README.adoc

```diff
@@ -16,7 +14,7 @@ Maven coordinates for the dialect:
 <dependency>
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-spanner-hibernate-dialect</artifactId>
-  <version> TBD </version>
+  <version>0.1.0.BUILD-SNAPSHOT</version>
```

Comment:
To use the snapshot version, does the user need to configure a special repository to achieve this?

Reply:
Not really, just Maven Central Snapshots.

Reply:
This is very temporary anyway.

Reply:
Can you mention this in the docs and then remove it once the official version is published?